Meta.svelte

```diff
@@ -3,15 +3,74 @@
   Metadata helper
 -->
 <script>
-  /** Page title */
+  /** Page title **/
   export let title: Optional<string> = undefined;
-  /** Page description */
+  /** Page description **/
   export let description: Optional<string> = undefined;
-  /** Page image */
+  /** Page image **/
   export let image: Optional<string> = undefined;
+  /** Page url **/
+  export let url: Optional<string> = undefined;
+  /** Site logo **/
+  export let logoUrl: string = url ? `${url}/logo.png` : undefined;
+  /** Search URL **/
+  export let searchUrl: Optional<string> = undefined;
+  /** Sitemap URL **/
+  export let sitemapUrl: string = url ? `${url}/sitemap.xml` : undefined;
+  /** Twitter Options **/
+  export let twitter: any = {
+    card: 'summary_large_image',
+    site: undefined,
+    creator: undefined,
+  };
+  /** OpenGraph Options **/
+  export let openGraph: any = {
+    type: undefined,
+    imageWidth: undefined,
+    imageHeight: undefined,
+    locale: 'en_US',
+    siteName: undefined,
+    imageAlt: title || undefined,
+  }
+  /** Facebook App ID **/
+  export let facebookAppId: Optional<string> = undefined;
 </script>
 
 <svelte:head>
+
+  <meta charset="utf-8" />
+  <meta http-equiv="x-ua-compatible" content="ie=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="robots" content="index,follow" />
+  <meta name="googlebot" content="index,follow" />
+  <link rel="sitemap" type="application/xml" href="{sitemapUrl}" />
+
+  <meta name="twitter:card" content="{twitter.card}" />
```

Comment:
Should be conditional rather than having a default? Otherwise could be setting things you're not aware you're setting

Reply:
Removing the default for now, but would be good to have enums for these.

Reply:
Yeah see above comments on impl deets for `twitter` prop, if we include it in this PR at all